README.md

```diff
@@ -461,6 +467,153 @@ However, there is one important restriction: you **must not decide to completely
 * **If single-site admins who cannot edit the dashboard should be able to edit it,** you should put it in `sites/modules/@apostrophecms/global`.
 * **If only dashboard admins who create and remove sites should be able to make this decision,** it belongs in `dashboard/modules/site/index.js`. You can then pass it on as module configuration in `sites/lib/index.js`.
 
+## Cypress (end-to-end) Testing 
+
+Cypress is configured as the default tool to run end-to-end tests on the multisite platform. The tests are located in the `cypress/test` folder. Read below to learn more about the prerequisites, how to run the tests, how to update your DB dump when needed, and how to integrate Cypress in your existing projects.
+
+The pre-configured experience includes a DB dump of the "dashboard" site, containing a site per theme ("default" and "demo"). By default, the tests will run against the "default" site theme. For testing the "demo" site theme and the "dashboard", your tests should override the `baseUrl` at the test file level and utilize the `profile` options on the relevant commands. See the `cypress/test` folder for examples.
+
+You can reconfigure the DB dump to include more or different sites and/or change the defaults. Read below to learn more.
+
+The multisite application will use `test-` prefixed collections in the MongoDB database to avoid losing local development data when running the tests. This is achieved by setting the `CI` environment variable to `1` when running the tests (see the `e2e:*` scripts in `package.json`).
+
+### Prerequisites
+
+Install [MongoDB Tools](https://docs.mongodb.com/database-tools/installation/installation-linux/#installation). You should validate that the `mongodump` and `mongorestore` commands are available in your terminal.
+
+### Running the tests
+
+You'll find example tests in the `cypress/test` folder. To run the tests:
+
+```bash
+# Start the multisite platform in production mode
+npm run e2e:serve
+# OR start the multisite platform in development mode (if you want to see the changes in real-time)
+npm run e2e:dev
+# In a new terminal window, run the tests in headless mode
+npm run e2e:run
+# OR run the tests in interactive mode
+npm run e2e:open
+```
+
+### Updating the Cypress configuration DB dumps 
+
+1. Remove all `test-` prefixed collections from your local MongoDB database.
```

Comment:
Maybe let users know they should do this using mongosh or compass?

Reply:
This is typo, it's not collections but "databases".